src/actions/configuration/first-time-configuration-action.php

```diff
@@ -295,4 +334,21 @@ public function get_configuration_state() {
 	private function can_edit_profile( $person_id ) {
 		return \current_user_can( 'edit_user', $person_id );
 	}
+
+	/**
+	 * Gets the old values for the given fields.
+	 *
+	 * @param array $fields_names The fields to get the old values for.
+	 *
+	 * @return array The old values.
+	 */
+	private function get_old_values( $fields_names ) : array {
```

Comment:
If you're adding the return type hinting, why not also the argument?

(then I noticed the filters and missing type checks 😅 -- still, might want to make those safer instead)

Reply:
Done.